.github/workflows/ci.yml

```diff
@@ -41,10 +59,9 @@ jobs:
       run: |
         git clone https://github.com/nextstrain/zika-tutorial
         pip install nextstrain-cli
-        nextstrain build --image nextstrain/base:$TAG zika-tutorial -F
+        nextstrain build --image localhost:5000/nextstrain/base:$TAG zika-tutorial -F
```

Comment:
Might be a good idea to run this test with `--platform linux/arm64`, though it'll be using emulation for the time being.

Reply:
Done: d4f91c4ecf7215a37fda54f19afdf3d08444b7ad